dist/formats/finder/frontend/schema.json

```diff
@@ -401,6 +401,10 @@
             "description": "The exact rummager field name for this facet. Allows 'key' to be aliased to a rummager filter field",
             "type": "string"
           },
+          "filter_value": {
+            "description": "A preset filter value that is applied when a checkbox is selected",
```

Comment:
A `filter_value` isn't necessarily for a checkbox facet.

Reply:
The checkbox is the only thing that reads this at the moment... I guess we might have other things later?  https://github.com/alphagov/finder-frontend/search?q=filter_value&unscoped_q=filter_value